Paracha_Junaid/Assignments/Flask_SQL/friends/templates/edit.html

```diff
@@ -0,0 +1,27 @@
+<html>
+  <head>
+    <title>Form Test Show</title>
+  </head>
+  <body>   
+    <h2>This is the Edit page </h2>
+    {%for val in users_edit:%}    
+    <form action="/friend/{{ val['id']}}" method="POST">
+      {{  val['User_Name']}}: <input type="text" name="email">
```

Comment:
The way this displays seems different.  Why not set the value attribute of the input to the value the user is editing?  That way they can see what it is and directly edit it.